libs/@local/hash-graph-sdk/typescript/src/entity.ts

```diff
@@ -103,6 +104,46 @@ const isGraphApiEntity = <Properties extends EntityPropertiesObject>(
   );
 };
 
+export const flattenedPropertyMetadataMap = (
```

Comment:
Suggest renaming this for two reasons:
1. It didn't sound like an action before, but more like the result (`flattened -> flatten`)
2. it's not producing what I would call a `Map`... I think fine to just drop the word
```suggestion
export const flattenPropertyMetadata = (
```